### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by lbakerchef**
Update release notes for 14.6.32

### CODE REVIEW

File: content/release_notes_server.md [hunk 1]
```diff
@@ -16,6 +16,61 @@ Chef Infra Server acts as a hub for configuration data by storing
...
+
+- CVE-2021-31810
+- CVE-2021-32066
+- CVE-2021-31799
+
+### Elasticsearch
+
+We've upgraded Elasticsearch from 6.8.16 to 6.8.17 to resolve CVE-2021-22144.
+
+
```
Comment (tas50) [inline content/release_notes_server.md]:
```suggestion
```

File: content/release_notes_server.md [hunk 2]
```diff
@@ -16,6 +16,61 @@ Chef Infra Server acts as a hub for configuration data by storing
...
+
```
Comment (tas50) [inline content/release_notes_server.md]:
```suggestion
```

**Comment by IanMadd** [comment]
1. The page title needs to be updated.
2. We need an H2 heading --> "What's New in 14.6.32"
3. All headings need to go down one, H2 --> H3, H3 --> H4, etc...

File: content/release_notes_server.md (hunk 2 above)
Comment (lbakerchef) [reply content/release_notes_server.md]: Done.

**Comment by lbakerchef** [comment]
> 1. The page title needs to be updated.
> 2. We need an H2 heading --> "What's New in 14.6.32"
> 3. All headings need to go down one, H2 --> H3, H3 --> H4, etc...

Done.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**